gear/gear/metrics.py

```diff
@@ -25,3 +28,38 @@ async def monitor_endpoints_middleware(request, handler):
         raise e
     finally:
         CONCURRENT_REQUESTS.labels(endpoint=endpoint, verb=verb).dec()
+
+
+class LoggingTimer:
```

Comment:
`LoggingTimer` is a bit of a misnomer now, perhaps `PrometheusSQLTimer`?

Reply:
I'm realizing now that the LoggingTimer is used in other places that don't have anything to do with monitoring SQL queries. I think we might have to design this a little differently to take care of the different use cases